lib/src/testing/utils/testrunner.dart

```diff
@@ -63,6 +63,9 @@ class _TestCase {
   // Parameters for key import (always required)
   final Map<String, dynamic>? importKeyParams;
 
+  // Parameters for import key exception (optional, if there is an exception)
+  final Map<String, dynamic>? importKeyException;
```

Comment:
Let's make this simpler.

```suggestion
  // If not `null`, then importing private or public key MUST throw
  // an exception that contains this string.
  final String? importKeyException;
```

Instead of making a test case that contains different strings for `pkcs8Exception` and `jwkException`, we just make two test cases. One that tests the exception when importing a PKCS8 key and on that tests it when importing a JWK key.

Notice that when specifying a test case, you don't have to specify both `privatePkcs8KeyData` and `privateJsonWebKeyData`, it's enough to specify one of them. So if we want to test different error messages (from different formats), we really should just make two test cases.

That way, we also avoid having an absolute explosion of properties on the test case objects.

Reply:
We also need to update `_validateTestCase`

Reply:
Updated with one string `importKeyException`. Made necessary changes to `_validateTestCase`